src/content/docs/2/guide/upgrade-migrate/from-tauri-1.mdx

```diff
@@ -0,0 +1,769 @@
+---
+title: Migrate from Tauri 1.0
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+
+This guide walks you through migrating your Tauri 1.0 application to Tauri 2.0.
+
+## Automated Migration
+
+The Tauri v2 CLI includes a `migrate` command that automates most of the process and helps you finish the migration:
```

Comment:
Can we explain what this command does so someone knows what to expect before running it? I need to get the CLI documenter up and running so we can also link to it.

Reply:
I think we should link to the command's help description instead. What it actually does will change a lot until v2. So far it migrates Cargo.toml and tauri.conf.json (that's mostly all it'll do, but we also want to guide the user on the plugins migration and some other breaking changes).

Reply:
Cool, I can rephrase that a bit and point to the new CLI docs that @FabianLars started in #1365 (will just be a broken link for now I guess).